core/src/main/java/org/apache/accumulo/core/spi/fs/RandomVolumeChooser.java

```diff
@@ -21,18 +21,30 @@
 import java.security.SecureRandom;
 import java.util.Set;
 
+import org.apache.accumulo.core.conf.Property;
+
 /**
+ * A {@link VolumeChooser} that selects a volume at random from the list of provided volumes. This
+ * class is currently the default volume chooser as set by {@link Property#GENERAL_VOLUME_CHOOSER}.
```

Comment:
```suggestion
 * A {@link VolumeChooser} that selects a volume at random from the list of provided volumes. 
```

If the default ever changes, then we have to remember to update this comment. Users can just check the default value of the property.

Reply:
Sounds good! Changes implemented